autoscheduler/frontend/src/package-lock.json

```diff
@@ -7,7 +7,7 @@
     "@babel/code-frame": {
       "version": "7.5.5",
       "resolved": "https://registry.npmjs.org/@babel/code-frame/-/code-frame-7.5.5.tgz",
-      "integrity": "sha512-27d4lZoomVyo51VegxI20xZPuSHusqbQag/ztrBC7wegWoQ1nLREPVSKSW8byhTlzTKyNE4ifaTA6lCp7JjpFw==",
+      "integrity": "sha1-vAeC9tafe31JUxIZaZuYj2aaj50=",
```

Comment:
This looks like it's reverting back to the old type of `npm` to me. When I run `npm install`, it changes the `integrity` to look like the line that was replaced.

Also, can you run `npm audit fix` and commit the changes? When I ran it locally it fixed the vulnerability just using that command. Over than that this should be good.

Reply:
Yeah, I noticed the messed-up integrities over in `frontend/css-modules`, so I'll just go ahead and apply the fix from that branch here before we merge into `frontend/master`. The end goal is to choose sha512 over sha1 whenever there's a conflict, and what I'm doing is `rm -rf node_modules && rm package-lock.json` followed by a fresh `npm install`. It worked in the other branch, but be forewarned, many packages will be updated in this process.

Reply:
Incidentally, Jest was also updated in this process, and `npm audit` now shows no issues